cli/sawtooth_cli/batch.py

```diff
@@ -17,6 +17,9 @@
 from sawtooth_cli import format_utils as fmt
 from sawtooth_cli.rest_client import RestClient
 from sawtooth_cli.exceptions import CliException
+from sawtooth_cli.parent_parsers import rest_parent
```

Comment:
These names are a little too generic, which made it difficult to figure out what they were for when reading the code.  

Suggest: `general_rest_args_parent`, `list_args_parent`, etc.